functional/kata-monitor/run.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/sh
+#
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test file will test kata-monitor for basic functionality (retrieve kata sandboxes)
+# It will assume an environment where:
+# - a CRI container manager (container engine) will be up and running
+# - crictl is installed and configured
+# - the kata-monitor binary is available on the host
+#
+
```

Comment:
I'd consider adding in the "boiler plate" code for consistency and to avoid surprises:

```bash
set -o errexit
set -o nounset
set -o pipefail
 
[ -n "$BASH_VERSION" ] && set -o errtrace

[ -n "${DEBUG:-}" ] && set -o xtrace
```

Reply:
yep, makes sense.
Will need to trap ERR than and change a bit the logic of error reporting to not lose that.